R/bmerge.R

```diff
@@ -89,13 +93,12 @@ bmerge = function(i, x, icols, xcols, roll, rollends, nomatch, mult, ops, verbos
     }
     cfl = c("character", "logical", "factor")
     if (x_merge_type %chin% cfl || i_merge_type %chin% cfl) {
-      msg = if(verbose) gettext("Coercing all-NA %s column %s to type %s to match type of %s.\n") else NULL
       if (anyNA(i[[icol]]) && allNA(i[[icol]])) {
-        coerce_col(i, icol, i_merge_type, x_merge_type, iname, xname, msg)
+        coerce_col(i, icol, i_merge_type, x_merge_type, iname, xname, gettext(" (all-NA)"), verbose = verbose)
         next
       }
       if (anyNA(x[[xcol]]) && allNA(x[[xcol]])) {
-        coerce_col(x, xcol, x_merge_type, i_merge_type, xname, iname, msg)
+        coerce_col(x, xcol, x_merge_type, i_merge_type, xname, iname, gettext(" (all-NA)"), verbose = verbose)
```

Comment:
maybe this is an error copied over from the code you're changing, but it looks wrong that both these cases use `from_detail="all-NA"` -- the first branch is `allNA(i[[icol]])`, the second is `allNA(x[[xcol]])`. I guess the latter case should be `to_detail="all-NA"`?

Reply:
Made this change in 7644d413e43faea6a6cd1860e6678fac85788390, CMIIW